content/intro-to-storybook/ember/en/deploy.md

```diff
@@ -0,0 +1,152 @@
+---
+title: 'Deploy Storybook'
+tocTitle: 'Deploy'
+description: 'Learn how to deploy Storybook online'
+---
+
+Throughout this tutorial, we built components on our local development machine. At some point, we'll need to share our work to get team feedback. Let's deploy Storybook online to help teammates review UI implementation.
+
+## Exporting as a static app
+
+To deploy Storybook we first need to export it as a static web app. This functionality is already built-in to Storybook and pre-configured.
+
+Running `npm run build-storybook`, it will output a static Storybook in the `storybook-static` directory, which can then be deployed to any static site hosting service.
```

Comment:
The Ember application must be build before. E.g. running `yarn build` if the app follows the latest blueprints. I think it should be mentioned explicitly as it's easy to miss that one. Struggled with it a lot when setting up my CI pipeline.

Reply:
@jelhan i'm aware of that. I'm still polishing some items for this pull request. And this is one of them. I'll update it shortly.